api/tests/functional-tests/backend/core/test_evaluation.py

```diff
@@ -421,13 +421,47 @@ def test_get_evaluations(
     assert evaluations_by_dataset_model_eval_id[0].id == created_2[0].id
 
     # make sure stratifying works by dataset and model
-    evaluations_by_dataset_model_eval_id = core.get_evaluations(
+    evaluations_by_dataset_model_eval_id, _ = core.get_evaluations(
         db=db,
         dataset_names=[finalized_dataset],
         model_names=[finalized_model],
     )
     assert len(evaluations_by_dataset_model_eval_id) == 2
 
+    # test pagination
+    with pytest.raises(ValueError):
+        # offset is greater than the number of items returned in query
+        evaluations, headers = core.get_evaluations(
+            db=db,
+            dataset_names=[finalized_dataset],
+            model_names=[finalized_model],
+            offset=6,
+            limit=1,
+        )
+
+    evaluations, headers = core.get_evaluations(
+        db=db,
+        dataset_names=[finalized_dataset],
+        model_names=[finalized_model],
+        offset=1,
+        limit=1,
+    )
+
+    assert len(evaluations) == 1
+    assert headers == {"content-range": "items 1-1/2"}
+
+    # check that having too high of a limit param doesn't throw an error
+    evaluations, headers = core.get_evaluations(
+        db=db,
+        dataset_names=[finalized_dataset],
+        model_names=[finalized_model],
+        offset=0,
+        limit=6,
+    )
+
+    assert len(evaluations) == 2
```

Comment:
Can you add a value check to show that the output is deterministic?

Reply:
see above; not sure what value you'd want to check

Reply:
See my comment below. I feel there needs to be a check that you actually get back all the same data if you combine multiple pages.